WebContent/WEB-INF/ftl/text/activeSms.ftl

```diff
@@ -0,0 +1,22 @@
+<#ftl strip_whitespace=false><#--
+    Mango - Open Source M2M - http://mango.serotoninsoftware.com
+    Copyright (C) 2006-2011 Serotonin Software Technologies Inc.
+    @author Matthew Lohbihler
+    
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see http://www.gnu.org/licenses/.
+-->
+<@fmt key="ftl.eventActive"/>
+
+<#include "include/eventData.ftl">
+<@fmt key="ftl.htmlFooter.mango"/>
```

Comment:
## Minor issue
It is not so important and can be omitted

Is this a complete solution? 
What about including a `<#include "include/footer.ftl">` line instead this highlighted one? Each *.ftl template use that `footer.ftl` template, especially **active.ftl** that is the file on which this one should be based on. The more similar are that files the more consistent the code is.